apps/pushy/src/pushy_zap.erl

```diff
@@ -107,12 +107,10 @@ code_change(_OldVsn, State, _Extra) ->
 %% @doc Add the client public key to the set of validated keys.
 %% of heartbeat intervals
 add_key_to_cache(Key, State = #state{clients = Clients}) ->
-    State#state{clients = dict:store(Key, time_to_integer(erlang:now()) + ?EXPIRATION_TIME, Clients)}.
-    
-time_to_integer({MegaS, S, _MicroS}) -> 1000000*MegaS + S.
+    State#state{clients = dict:store(Key, erlang:system_time(seconds) + ?EXPIRATION_TIME, Clients)}.
 
 expire_clients(State = #state{clients = Clients}) ->
-    Now = time_to_integer(erlang:now()),
+    Now = erlang:system_time(seconds),
```

Comment:
Here we were calling time_to_integer to get seconds, so I've gone with just getting seconds directly.